### ISSUE TITLE
crc setup loop

### ISSUE THREAD

**Issue by cbeech1980** [code-ready/crc#2540]
Hi all,

Trying to setup crc on Windows 10 Pro [Version 10.0.19041.1052] and am having an issue with the crc setup command. 

When I run it I get the following output...

```
C:\Users\ChrisBeech>crc setup
INFO Checking if crc-admin-helper executable is cached
INFO Checking for obsolete admin-helper executable
INFO Checking if running on a supported CPU architecture
INFO Checking minimum RAM requirements
INFO Checking if running in a shell with administrator rights
INFO Checking Windows 10 release
INFO Checking Windows edition
INFO Checking if Hyper-V is installed and operational
INFO Checking if crc-users group exists
INFO Checking if current user is in Hyper-V group and crc-users group
INFO Adding current user to group
INFO Will run as admin: adding current user to crc-users group
Please reboot your system and run 'crc setup' to complete the setup process
```

...the problem is that when I reboot and run it again, it gives me exactly the same output!

I'm going round in circles and can't seem to figure out why.

Any ideas?

Chris

**Comment by gbraad** [comment]
Is your machine domain joined?

**Comment by cbeech1980** [comment]
My user account is an Azure AD account, so it's joined to AAD.

**Comment by anjannath** [comment]
Duplicate of #2446 and was fixed in #2494 , we'll have a new release this week which will have that fix.

**Comment by cfergeau** [comment]
Closing this since it's a duplicate.
Duplicate of #2446
